providers/ms365/resources/ms365.lr

```diff
@@ -163,21 +163,72 @@ private microsoft.domaindnsrecord @defaults("id label") {
 }
 
 // Microsoft application
-private microsoft.application @defaults("id displayName") {
+microsoft.application @defaults("id displayName hasExpiredCredentials") {
+  init(name string)
   // Application ID
   id string
   // Application app ID
   appId string
+  // Application display name
+  name string
+  // Deprecated: Application display name use `name` instead
+  displayName string
+  // Description
+  description string
+  // Notes
+  notes string
+  // Tags
+  tags []string
   // Application creation date
+  createdAt time
+  // Deprecated: Application creation date use `createdAt` instead
   createdDateTime time
   // Application identifier URIs
   identifierUris []string
-  // Application display name
-  displayName string
   // Application publisher domain
   publisherDomain string
   // Application sign-in audience
   signInAudience string
+  // Basic profile information
+  info dict
+  // Client secrets
+  secrets []microsoft.passwordCredential
+  // Certificates
+  certificates []microsoft.keyCredential
+  // Indicates assigned expired credentials
+  hasExpiredCredentials() bool
+}
+
+// Certificate Secret
```

Comment:
Since these all end up being the titles of the docs pages is there a more descriptive name we can use for each of these that combines the technology + specific item?